[next] Add a packagegroup for SNAC

amstewart opened the pull request:
Add a packagegroup to track package dependencies of the NILRT Secured,
Network-Attached Controller (SNAC) configuration.

Since the SNAC configuration is officially supported, add it to the core
package feed.

This packagegroup SHOULD NOT be installed to the NILRT runmode or
safemode images.


# Testing
* [x] Built `packagegroup-ni-snac` with this change. Meta package seems well formed.
* [x] Ran `bitbake -g packagefeed-ni-core` with this change and confirmed that the new packagegroup is in the pn-buildlist.

# Meta
* This is a NILRT 11 change only.

Commit:
packagegroups: add pkggrp-ni-snac

Add a packagegroup to track package dependencies of the NILRT Secured,
Network-Attached Controller (SNAC) configuration.

Since the SNAC configuration is officially supported, add it to the core
package feed.

This packagegroup SHOULD NOT be installed to the NILRT runmode or
safemode images.

Signed-off-by: Alex Stewart <astewart.49c6@gmail.com>

Commit:
pkggrp-ni-snac: add initial top level deps

Add some initial packages that we know are going to be used in the SNAC
v1.0 design.

Signed-off-by: Alex Stewart <astewart.49c6@gmail.com>

amstewart commented on recipes-core/packagegroups/packagegroup-ni-snac.bb:
```diff
@@ -0,0 +1,10 @@
+SUMMARY = "Open source package dependencies for the NILRT SNAC configuration."
+LICENSE = "MIT"
+
```
Forgot the packagegroup inherit. :) Will fix.

amstewart commented:
# Patch V2
* Fixed missing inherit.

chaitu236 commented on recipes-core/packagegroups/packagegroup-ni-snac.bb:
```diff
@@ -0,0 +1,14 @@
+
+RDEPENDS:${PN} = ""
+
+RDEPENDS:${PN}:append = "\
```
Why list these in `RDEPENDS:${PN}:append` and not in `RDEPENDS:${PN}`?

amstewart replied:
I guess force of habit - since these packagegroups normally have many subsections. But I guess it doesn't make sense in this case. Will fix.

amstewart replied:
Fixed.

Commit:
pkggrp-ni-snac: remove unneeded append assignment

The snac packagegroup is only expected to contain one logical subsection
of packages, so there is no need to split up the RDEPENDS assignments.
Use a single section.

Signed-off-by: Alex Stewart <astewart.49c6@gmail.com>
